examples/accessing_tiles.rs

```diff
@@ -160,9 +160,9 @@ fn main() {
                 })
                 .set(ImagePlugin::default_nearest()),
         )
-        .add_plugin(TilemapPlugin)
-        .add_startup_system(startup)
-        .add_system(helpers::camera::movement)
-        .add_system(update_map)
+        .add_plugins(TilemapPlugin)
+        .add_systems(Startup, startup)
+        .add_systems(Update, helpers::camera::movement)
+        .add_systems(Update, update_map)
```

Comment:
These calls can be combined. The `add_systems` method allows you to add multiple systems in one call.